src/bin/import_lines.rs

```diff
@@ -36,30 +36,15 @@ fn main() {
 
     if opt.producer.starts_with('Q') {
         info!("Searching the producer by id");
-        match client.sparql.find_producer(&opt.producer) {
```

Comment:
To be discussed, but we no longer check that the producer actually is a producer. A typo, and *poof* the data will be produced by a trolleybus :trolleybus: